packages/styles/scss/patterns/sub-patterns/card/index.scss

```diff
@@ -25,6 +25,7 @@
       flex-direction: column;
       padding: $carbon--spacing-05;
       min-height: carbon--mini-units(20);
+      max-width: 90%;
```

Comment:
Don't we have a utility for this? Also, I would add this to the copy and heading instead of the wrapper.

Reply:
@photodow do you mean `max-width`? We have no utility for that, that I know of.